chain-abci/src/staking/table.rs

```diff
@@ -190,19 +190,18 @@ impl StakingTable {
     }
 
     /// Handle reward statistics record
-    /// TODO change to vote statistics
     pub fn reward_record(
         &mut self,
         heap: &impl GetStaking,
         val_addr: &TendermintValidatorAddress,
     ) -> bool {
         if let Some(addr) = self.idx_validator_address.get(val_addr) {
-            // Invarient 2.1
+            // Invariant 2.1
             let staking = heap.get(addr).unwrap();
-            // Invarient 2.2
+            // Invariant 2.2
             let val = staking.validator.as_ref().unwrap();
             if val.is_active() {
-                self.proposer_stats
+                self.participator_stats
                     .entry(*addr)
                     .and_modify(|count| *count = count.saturating_add(1))
```

Comment:
Should record the sum of voting powers, so the distribution is proportional to it.